source/globalCommands.py

```diff
@@ -1184,6 +1184,29 @@ def script_review_endOfLine(self,gesture):
 	script_review_endOfLine.__doc__=_("Moves the review cursor to the last character of the line where it is situated in the current navigator object and speaks it")
 	script_review_endOfLine.category=SCRCAT_TEXTREVIEW
 
+	def script_review_currentSymbol(self,gesture):
+		info=api.getReviewPosition().copy()
+		info.expand(textInfos.UNIT_CHARACTER)
+		text = info.text
+		curLanguage = None
+		if config.conf['speech']['autoLanguageSwitching']:
+			for field in info.getTextWithFields({}):
+				if isinstance(field,textInfos.FieldCommand) and field.command=="formatChange":
+					curLanguage=field.field.get('language')
+		if curLanguage is None:
+			curLanguage = speech.getCurrentLanguage()
+		expandedSymbol = characterProcessing.processSpeechSymbol(curLanguage, text)
+		repeats=scriptHandler.getLastScriptRepeatCount()
+		if repeats == 0:
+			# Explicitly tether here

```

Comment:
Rather than tethering and using speech.SpeakTextInfo, I think I'd prefer just calling ui.message(expandedSymbol)